Content.Client/Access/UI/IdCardConsoleWindow.xaml.cs

```diff
@@ -55,7 +55,7 @@ public sealed partial class IdCardConsoleWindow : DefaultWindow
 
             foreach (var job in jobs)
             {
-                if (!job.SetPreference)
+                if (!job.ShowInIdCardConsole)
```

Comment:
change this to be an OR statement with the previous condition so you don't have to update the other prototypes. Both conditions are perfectly valid.

Reply:
I double-checked this and it appears to not work;

With an OR statement, you fail one of these conditions:
Job selectable in lobby, and in console: Works as intended
Job selectable in lobby, but not in console: Works as intended
Job not selectable in lobby, and not in console: Works as intended
Job not selectable in lobby, but in console: Does not work, job doesn't show up in the console.

As the default value of ShowInCardConsole is true, there'd be no way to set the Visitor job to appear in the console but not as a selectable job.

Reply:
OR statement would not work; the Visitor job role would then not be shown as its SetPreference is false.

Instead I changed `ShowInCardConsole` to `OverrideConsoleVisibility`, so that if set it ensures the job does/doesn't show up. If it's not set, it defaults to the SetPreference value.

Reply:
This also allows making a role that can be set in the lobby but doesn't show up as a console job, if one would want to for some reason.